src/hotspot/share/opto/bytecodeInfo.cpp

```diff
@@ -313,10 +313,10 @@ bool InlineTree::should_not_inline(ciMethod* callee_method, ciMethod* caller_met
     if (MinInlineFrequencyRatio > 0) {
       int call_site_count  = caller_method->scale_count(profile.count());
       int invoke_count     = caller_method->interpreter_invocation_count();
-      assert(invoke_count != 0, "require invocation count greater than zero");
+      assert(invoke_count >= 0, "require invocation count greater than zero");
```

Comment:
Technically, the comment is now wrong. It is no longer "greater than" but "greater than or equal to zero". Is that intended? Otherwise you should use `>`.

Reply:
Done.